make-dist.sh

```diff
@@ -25,7 +56,16 @@ mkdir -p "$DIST_DIR/admin-dist"
 
 cp server/szuru-admin "$DIST_DIR/admin-dist/szuru-admin"
 cp server/szuru_admin_argparse.py "$DIST_DIR/admin-dist/szuru_admin_argparse.py"
-cp server/szuru_admin_version.py "$DIST_DIR/admin-dist/szuru_admin_version.py"
+
```

Comment:
Rather than removing the cp, do the cp as normal and then edit the version in the dest file.